buildchain/buildchain/packaging.py

```diff
@@ -346,6 +339,61 @@ def _deb_package(name: str, sources: Path) -> targets.DEBPackage:
     task_dep=['_package_mkdir_rpm_iso_root'],
 )
 
+# }}}
+# Debian packages and repositories {{{
+
+def _deb_package(name: str, sources: Path) -> targets.DEBPackage:
+    try:
+        pkg_info = versions.RPM_PACKAGES_MAP[name]
```

Comment:
`_deb_package` using `RPM_PACKAGES_MAP`?

Reply:
Oh boy that's bad.